public/app/plugins/datasource/grafana-azure-monitor-datasource/resourcePicker/resourcePickerData.ts

```diff
@@ -30,9 +32,9 @@ export default class ResourcePickerData {
   private proxyUrl: string;
   private cloud: string;
 
-  constructor(proxyUrl: string, cloud: string) {
-    this.proxyUrl = proxyUrl;
-    this.cloud = cloud;
+  constructor(instanceSettings: DataSourceInstanceSettings<AzureDataSourceJsonData>) {
```

Comment:
nit: `AzureDataSourceInstanceSettings` is alias for `DataSourceInstanceSettings<AzureDataSourceJsonData>`